src/routes/privacy-policy.svelte

```diff
@@ -17,63 +17,135 @@
     <ul>
       <li>
-        Delete all cookies
-      </li>
-      <li>
-        Block all cookies
-      </li>
-      <li>
-        Allow all cookies
-      </li>
-      <li>
-        Block third-party cookies
-      </li>
-      <li>
-        Clear all cookies when you close the browser
+        <b>Hotjar</b>
+        <p>We use Hotjar to better understand your needs and to optimise this service and experience. Hotjar is a technology service that helps us better understand your experience (e.g., how much time you spend on which pages, which links you choose to click, what you do and don’t like, etc.) and this enables us to build and maintain our service with user feedback. Hotjar uses cookies and other technologies to collect data on your behaviour and devices. This includes a device's IP address (processed during your session and stored in a de-identified form), device screen size, device type (unique device identifiers), browser information, geographic location (country only), and the preferred language used to display our website. Hotjar stores this information on our behalf in a pseudonymised (de-identified) user profile. Hotjar is contractually forbidden to sell any of the data collected on our behalf.</p>
+        <p>If you interact with our surveys or feedback forms provided by Hotjar, additional cookies are set to support that service. Data you submit is used only for the purpose stated and governed by our data handling policies detailed below. We only collect Personally Identifiable Information through Hotjar when submitted voluntarily on surveys or feedback forms. Otherwise, Hotjar analytics does not collect Personally Identifiable Information when tracking usage.</p>
+        <p>For further details, please see the ‘about Hotjar’ section of <a href="https://help.hotjar.com/hc/en-us/categories/115001323967-About-Hotjar" target="_blank">Hotjar’s support site</a>.</p>
       </li>
       <li>
-        Open a 'private browsing' / 'incognito' session (allows you to browse the internet without storing local data)
+        <b>Google Analytics</b>
+        <p>We use Google Analytics to monitor usage of the CPR websites and apps. This information allows us to assess how our websites and apps are used. We collect and aggregate anonymous data about usage (e.g., number of visitors, visitors' country, number of times each page is visited) to produce performance reports.</p>
+        <p>You may opt‐out of Google Analytics by installing the Google Analytics opt‐out browser add‐on or by configuring your browser settings to block the use of cookies.</p>
+        <p>For further details, please see the ‘Analytics’ section of Google’s <a href="https://policies.google.com/technologies/cookies?hl=en-US" target="_blank">Privacy Policy</a>.</p>
```

Comment:
I noticed there is a difference between "...hl=en-US" and "...hl=en-UK" .. are we sure we are pointing to the right one? Not even sure what the right one is :shrug: /cc @alanwrightcpr

Reply:
Maybe just remove the param :grin: